docs/supported-libraries.md

```diff
@@ -19,116 +19,116 @@ These are the supported libraries and frameworks:
-| [Apache HttpClient](https://hc.apache.org/index.html)                                                                                       | 2.0+                          | [opentelemetry-apache-httpclient-4.3](../instrumentation/apache-httpclient/apache-httpclient-4.3/library)                                                                                                                                                                                                                                                                               |                                                                                        |
+| [Akka Actors](https://doc.akka.io/docs/akka/current/typed/index.html)                                                                       | 2.5+                          | N/A                                                                                                                                                                                                                                                                                                                                                                                     | none                                                                                   |
+| [Akka HTTP](https://doc.akka.io/docs/akka-http/current/index.html)                                                                          | 10.0+                         | N/A                                                                                                                                                                                                                                                                                                                                                                                     | [HTTP Client Spans], [HTTP Client Metrics], [HTTP Server Spans], [HTTP Server Metrics] |
+| [Apache Axis2](https://axis.apache.org/axis2/java/core/)                                                                                    | 1.6+                          | N/A                                                                                                                                                                                                                                                                                                                                                                                     | see Servlet                                                                            |
```

Comment:
it's not clear to me what this means?

maybe something like `server span name and controller span`?

Reply:
I was trying to reflect the idea the idea of layered instrumentations: Axis2 contributes additional context to the [servlet](https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/servlet) instrumentation. 

Might have gotten some of these wrong as it wasn't 100% clear to me how instrumentations interacted with one another.

Do you think `server span name and controller span` sufficiently tells the story of what kind of instrumentation you can expect from these frameworks?

Reply:
Instead of mentioning Servlet, maybe something like:

* provides route-based span name for existing `SERVER` span
* provides `http.route` span and metric attribute on existing `SERVER` span and metrics

since several of these also can be used with Netty (and further complicated because often the "servlet" spans are captured by native server instrumentation, e.g. jetty, before they even reach the servlet layer)

for controller spans (and view spans), a link to
https://opentelemetry.io/docs/instrumentation/java/automatic/agent-config/#suppressing-controller-andor-view-spans may be helpful

Reply:
I've taken a stab at fixing this